common/amount.c

```diff
@@ -401,22 +401,20 @@ bool amount_msat_to_u32(struct amount_msat msat, u32 *millisatoshis)
 	return true;
 }
 
-void amount_msat_from_u64(struct amount_msat *msat, u64 millisatoshis)
+struct amount_msat amount_msat(u64 millisatoshis)
```

Comment:
I like this better than the `void foo(out* out, int input)` style.  This is in fact what I thought `AMOUNT_SAT`/`AMOUNT_MSAT` did back when they were introduced, and was disappointed to learn they were just constexpr, boo.

The compiler might be able to optimize these better if we `static inline` these in the headers.